.babelrc

```diff
@@ -1,4 +1,9 @@
 {
   "presets": ["env", "react"],
-  "plugins": ["transform-object-assign", "transform-object-rest-spread", "transform-async-to-generator"],
+  "plugins": [
+    "transform-object-assign",
+    "transform-object-rest-spread",
+    "transform-async-to-generator",
+    "transform-class-properties"
```

Comment:
Should we be scoping these changes to the date-picker project as well?

https://babeljs.io/docs/en/config-files#monorepos

Reply:
https://github.com/cerner/terra-framework/pull/573/commits/b65c948b8bb448c5411e1e8ac76b65a387da50f5